README.md

```diff
@@ -10,6 +10,42 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
```

Comment:
I don't know it's a style guide I think file structure can be part of that, it's an art to organize your files